Properly disable the use of the xglk library if the port's XGLK option
has been turned off.

--- source/Makefile.orig
+++ source/Makefile
@@ -73,8 +73,9 @@
 # If your make can't handle sinclude or -include, and you're not building
 # for Glk, comment out the include line.
 GLKPATH	= $(GLKDIR)/$(GLK)
-sinclude $(GLKPATH)/Make.$(GLK)
-GLKLIBS	= -L$(GLKPATH) $(GLKLIB) $(LINKLIBS)
+#sinclude $(GLKPATH)/Make.$(GLK)
+#GLKLIBS	= -L$(GLKPATH) $(GLKLIB) $(LINKLIBS)
+GLKLIBS	=
 
 # Always use the Bourne shell.
 SHELL	= /bin/sh
